test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake
+import os
+
+class TypeSafe(ConanFile):
+    settings = 'os', 'compiler', 'build_type', 'arch'
+    requires = (
+        'cmake-utils/0.0.0@Manu343726/testing',
+        'type_safe/0.1@Manu343726/testing'
+    )
+    generators = 'cmake'
+
+    def build(self):
+        cmake = CMake(self.settings)
+        self.run('cmake {} {}'.format(self.conanfile_directory, cmake.command_line))
```

Comment:
new ``cmake.configure()`` and ``cmake.build()`` methods might be more convenient.